en/deploy-tidb-from-kubernetes-gke.md

```diff
@@ -44,14 +44,10 @@ This tutorial requires use of the Compute and Container APIs. Please enable them
 
 This step defaults gcloud to your preferred project and [zone](https://cloud.google.com/compute/docs/regions-zones/), which simplifies the commands used for the rest of this tutorial:
 
-{{< copyable "shell-regular" >}}
```

Comment:
This is required in the official doc website, please help confirm. @ran-huang

Reply:
This document is not meant for use/deployment on the official website, it's meant to be used in the Google Cloud Shell.